java/test/src/main/java/test/Ice/packagemd/NoPackage.ice

```diff
@@ -2,7 +2,9 @@
 
```

Comment:
The `packagemd` test is explicitly about testing `java:package` metadata.
It was left alone, except for adding some warning suppression, and making sure we test applying it to a module as well as at file level.